src/app/services/ConnectedUserService.ts

```diff
@@ -169,6 +169,8 @@ export class ConnectedUserService implements OnDestroy {
         try {
             const userCredential: FireAuth.UserCredential =
                 await Auth.createUserWithEmailAndPassword(this.auth, email, password);
+            // Directly logs in
+            await Auth.signInWithEmailAndPassword(this.auth, email, password);
```

Comment:
c'est pas déjà le cas en prod ?

Reply:
nein me semble que le processus était différent, maintenant c'est nécéssaire de te logger pour créer ton user. Avant ça t'enregistrait, puis tu créais un user anonymement (!), mais c'est plus autorisé pour la sécu.

Reply:
nécessaire de te logger pour nommer ton user, tu veux dire, non?

Reply:
aussi pour créer ton user en db (allow create is userIsVerified)